tuxemon/event/actions/random_encounter.py

```diff
@@ -124,7 +124,6 @@ def update(self) -> None:
             self.stop()
 
     def cleanup(self) -> None:
-        npc = None
```

Comment:
This code is changed in #1117, see https://github.com/Tuxemon/Tuxemon/pull/1117/commits/5de6f9e532ff2b8a6d225876402f8281e76d5b2f.

This PR is needed as it stops the game from crashing on random encounters.